css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,21 @@
+button {
+  padding: 10px 25px;
+}
+
+button:hover {
+  background-color: black;
+  color: white;
+}
```

Comment:
The pointer should also change on hover:

---

![image](https://user-images.githubusercontent.com/16143771/158652616-03577676-8a30-4c81-af6c-0b37a333d508.png)

Reply:
Fixed by adding 
cursor: pointer;
in the css rule for the button:hover selector